src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -2146,8 +2146,13 @@ uint os::processor_id() {
   assert(processor_id >= 0 && processor_id < os::processor_count(), "invalid processor id");
 
   return (uint)processor_id;
-}
+#else
```

Comment:
Please add a comment indicating what platform this else is referring to.

Reply:
Will fix!